src/TaskContext.js

```diff
@@ -21,18 +21,16 @@ export function TaskProvider({ children }) {
 
   const removeTask = (newTasks, boardId, id) => {
```

Comment:
When writing functions, follow this structure (till you find a better one :joy:)

```
/*
* @param newTasks: the task to be removed
* @param boardId: the board the task needs to be removed from
* @param id: the id of the task to be removed
*
* @returns: if something is returned, write that here
* 
* (next explain in one line what the function does)
* removes a task from a specific board
* 
* (in some cases provide some examples)
*/
const removeTask = (newTasks, boardId, id) => 
``` 

Just remember this structure the `next time` you write a function.